utils/uledd/Makefile

```diff
@@ -8,7 +8,7 @@ PKG_SOURCE_PROTO:=git
 PKG_SOURCE_VERSION:=bc94c7e52009f7ec33b23652502b4754351d8040
 PKG_MIRROR_HASH:=23a407197d6a423bb0b32f381e93ad3676e15f07465d7b53c843a2b510c7c284
 PKG_MAINTAINER:=John Crispin <john@phrozen.org>
-PKG_LICENSE:=GPL-2.0
+PKG_LICENSE:=LGPL-2.1-only
 
 CMAKE_INSTALL:=1
```

Comment:
I would remove this. PKG_INSTALL is already default because of cmake.mk.

CMAKE_INSTALL is PKG_INSTALL + InstallDev. There's no need to install the binary to InstallDev where it cannot be ran.

Reply:
Maybe add PKG_BUILD_PARALLEL:=1 as well.

Reply:
Thanks for the suggestions, but I fail to see how this is related to the scope of this PR.